Peoplesoft/variables.tf

```diff
@@ -5,8 +5,18 @@ variable "subscription_id" {
     description = "Azure Subscription GUID for the provisioning subscription. e.g., 666988bf-86f1-43af-91ab-2d7cd011db47"
 }
 
+variable "client_secret" {
+  
+}
+
+variable "client_id" {
+}
+variable "vnet_resource_group_name" {
+    default = "ps-rg3"
+    description = "Name of the Resource Group where the VNET is/will be located. This can be same as the app resource group or different."
```

Comment:
Are you using this variable? This variable is for those times when your VNET and GW may be in a different Resource group as the app RG

Reply:
Fixed.